### ISSUE TITLE
Issue adding secondary utility nav to mobile menu

### ISSUE THREAD

**Issue by pinksharpii** [FrDH/jQuery.mmenu#708]
The documentation for this plugin is pretty hard to follow and not up to date with the repo. I'm trying to add a utility nav that's on our client site to the menu (so that it displays in addition to the main menu items). 

I tried doing something like:
`$("#mobilemenu").mmenu({
	navbars:  [
		{
			position	: 'top',
			content	: $('#top-nav > ul')
		}
	]
});`

But it did not affect the menu. It's confusing because the demo file has "navbars" as an option but it's not outlined in the documentation.

I thought maybe tapping into the API I could manually insert the utility nav to the menu. I assume "initPanels" is the callback when it's done loading and I can add to it? It's not documented. And nothing happened when I tried this either.

How can I add a second navigation menu to the mobile menu?

**Comment by FrDH** [comment]
> I'm trying to add a utility nav that's on our client site to the menu

I'm not sure I understand what you're trying to achive. Is there an online example I could have a look at?


> it's not outlined in the documentation.

Well it is, but I understand it might be a bit hard to find.
From the [options page](http://mmenu.frebsite.nl/documentation/options/):
> If you're looking for the options for an add-on, check the add-ons pages.

So you should have a look at the [navbars add-on page](http://mmenu.frebsite.nl/documentation/addons/navbars.html).

Using `content: $("a-jquery-selector")` should basically work, but you'll end up with adding an entire UL to the navbar. My guess is you need to change `#top-nav > ul` into `#top-nav > ul a` or something similar to target the actual anchors.

Hope this helps.
